cylc/flow/xtrigger_mgr.py

```diff
@@ -770,11 +769,64 @@ def callback(self, ctx: 'SubFuncContext'):
 
         LOG.debug('%s: returned %s', sig, results)
         if not satisfied:
+            LOG.info('xtrigger not satisfied: %s = %s', ctx.label, sig)
```

Comment:
This logging would be more helpful to users if they could see which tasks are depending on this xtrigger.

Reply:
That could be a very long list.

I don't think we should be logging this at the INFO level at all. The number of unsuccessful xtrigger polls is not of interest to the user or developer.

Reply:
We could, but then what would be the point of doing it in the first place.

And again, I really don't think we should log this (at INFO level) at all.